SmartDeviceLink/SDLError.m

```diff
@@ -128,8 +129,8 @@ + (NSError *)sdl_lifecycle_startedWithBadResult:(SDLResult)result info:(NSString
 
 + (NSError *)sdl_lifecycle_startedWithWarning:(SDLResult)result info:(NSString *)info {
     NSDictionary<NSString *, NSString *> *userInfo = @{
```

Comment:
Tests are failing here when `result` is `nil`. 
> -[__NSPlaceholderDictionary initWithObjects:forKeys:count:]: attempt to insert nil object from objects[0]"

Reply:
Fixed